core/sql/sqlcomp/CmpSeabaseDDLcommon.cpp

```diff
@@ -3422,7 +3440,7 @@ Int64 CmpSeabaseDDL::getObjectUIDandOwners(
     strcpy(cfvd, " and valid_def = 'Y' ");
 
   char buf[4000];
```

Comment:
May consider moving this array to heap, because of the size of the array. Stack overflow prevention.